.azure-pipelines/ci-build.yml

```diff
@@ -94,6 +94,13 @@ steps:
     configuration: Release
     msbuildArguments: '/t:pack /p:PackageOutputPath=$(Build.ArtifactStagingDirectory) /p:IncludeSymbols=true /p:SymbolPackageFormat=snupkg'
 
+- task: MSBuild@1
+  displayName: 'Pack OpenAPI.Tool'
+  inputs:
+    solution: src/Microsoft.OpenApi.Tool/Microsoft.OpenApi.Tool.csproj
```

Comment:
The NuGet package will be named Microsoft.OpenApi.Tool. Is there a more specific name for this tool?

Reply:
From my conversation with @darrelmiller, we're still brainstorming whether to retain it as is or find a more descriptive name for the tool.

Reply:
Maybe `OpenApiCli.Tool` @darrelmiller ?

Reply:
I'm very close to being convinced we should call this Microsoft.hidi   as hidi is the Swahili word for Convert.

Reply:
Do either of you foresee this tool with future functionality that does something other than convert and validate? If this is foreseeable, then perhaps we leave it as is.

Reply:
Yes, it is going to slice.  It may also convert from CSDL to OpenAPI.  It may in the future be able to apply overlays.  I ran the name "hidi" up the chain and we have support from PMs.

Reply:
So then, `Microsoft.OpenApi.Hidi` or `Microsoft.Hidi`? I'm more for the first, but think both are fine.

Reply:
I was planning the second.  However, I have been asked to chat with the marketing folks to make sure they are ok with the hidi and kiota names. I'm going try and do that tomorrow.

Reply:
We can merge the PR and get approval before we push a release out.